app/controllers/space_contacts_controller.rb

```diff
@@ -0,0 +1,55 @@
+# frozen_string_literal: true
+
+class SpaceContactsController < AuthenticateController
+  before_action :set_space_contact, except: [:create]
+
+  def create
+    @space_contact = SpaceContact.create!(space_contact_params)
+
+    if @space_contact.save
+      redirect_to space_path(@space_contact.space_id), notice: t('space_contacts.contact_created')
```

Comment:
Because the form that calls this is outside of a turbo frame, this has the effect of reloading the page when all we wanted to do was to hide the form and show the newly updated using turbo stream. 

I think the "create new" form should be updated to correctly use turbo, but that can perhaps be done in a seperate PR?

Reply:
It is updated now - Mathias discovered a bigger bug that I was not aware on regarding turbo. Now it, should at least, work as excpected.